tests/drivers/gpio/gpio_basic_api/CMakeLists.txt

```
@@ -1,6 +1,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 cmake_minimum_required(VERSION 3.13.1)
+
+# This test requires devicetree fixture support
+set(DTS_EXTRA_CPPFLAGS "-DDTS_INCLUDE_FIXTURES")
```

Please do this as a DTS_OVERLAY_FILE

```suggestion
if(EXISTS ${ZEPHYR_BASE}/tests/common/${BOARD}/gpio-fixture.overlay)
  list(APPEND DTC_OVERLAY_FILE ${ZEPHYR_BASE}/tests/common/${BOARD}/gpio-fixture.overlay)
endif()